### PR TITLE
Add Heroku instructions to sessions documentation

### PR DESCRIPTION
**Pull request by tsujp**
- Add documentation to `sessions.rst` which explains how to use Telethon with Heroku if SQL versions are out of sync regarding support for `WITHOUT ROWID`.

### CODE REVIEW

File: readthedocs/extra/advanced-usage/sessions.rst [hunk 1]
```diff
@@ -44,3 +44,53 @@ methods. For example, you could save it on a database:
 
 You should read the ````session.py```` source file to know what "relevant
 data" you need to keep track of.
+
+
+Sessions and Heroku
+-------------------
+
+You probably have a newer version of SQLite installed (>= 3.8.2). Heroku uses
+SQLite 3.7.9 which does not support `WITHOUT ROWID`. So, If you generated your
```
Comment (Lonami) [inline readthedocs/extra/advanced-usage/sessions.rst]: RST uses double tick (\`\`) for code.

File: readthedocs/extra/advanced-usage/sessions.rst [hunk 2]
```diff
@@ -44,3 +44,53 @@ methods. For example, you could save it on a database:
...
+
+There are multiple ways to solve this, the easiest of which is generating a
+session file on your Heroku dyno itself. The most complicated is creating
+a custom buildpack to install SQLite >= 3.8.2.
+
+
+Generating Session File on a Heroku Dyno
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Do not restart your application Dyno at any point prior to retrieving your
```
Comment (Lonami) [inline readthedocs/extra/advanced-usage/sessions.rst]: You could use a ``.. note::` or even `.. warning::` block. Makes more sense than all bold (see other files for usage examples).

File: readthedocs/extra/advanced-usage/sessions.rst [hunk 3]
```diff
@@ -44,3 +44,53 @@ methods. For example, you could save it on a database:
...
+Due to Heroku's ephemeral filesystem all dynamically generated
+files not part of your applications buildpack or codebase are destroyed upon
+each restart.
+
+Using this scaffolded code we can start the authentication process:
+
+    .. code-block:: python
+
+        client = TelegramClient('login.session', api_id, api_hash,
+                                update_workers = 1, spawn_read_thread = False)
```
Comment (Lonami) [inline readthedocs/extra/advanced-usage/sessions.rst]: Prefer leaving no spaces when setting named parameters (like `update_workers=1`).

File: readthedocs/extra/advanced-usage/sessions.rst [hunk 4]
```diff
@@ -44,3 +44,53 @@ methods. For example, you could save it on a database:
...
+        client.start()
+
+At this point your Dyno will crash because you cannot access stdin. Open your
+Dyno's control panel on the Heroku website and "run console" from the "More"
+dropdown at the top right. Enter `bash` and wait for it to load.
```
Comment (Lonami) [inline readthedocs/extra/advanced-usage/sessions.rst]: Same here needs \`\` instead a single \`.

**Comment by tsujp** [comment]
I pushed a new commit with the changes you requested @Lonami.

**Comment by Lonami** [comment]
Not bothering you more still a tiny few things I changed. All good now, thanks!

**Pull request merged**